cmd/finish_image_build.go

```diff
@@ -93,13 +96,34 @@ func (f *FinishImageBuild) SetFlags(flags *flag.FlagSet) {
 	flags.BoolVar(&f.inheritLabels, "inherit-labels", false, "Indicates if the result image should inherit labels "+
 		"from the source image. Labels specified through the '-labels' flag take precedence over inherited "+
 		"labels.")
+	flags.StringVar(&f.oemSize, "oem-size", "", "Size of the new OEM partition, "+
+		"can be a number with unit like 10G, 10M, 10K or 10B, "+
+		"or without unit indicating the number of 512B sectors.")
 	flags.IntVar(&f.diskSize, "disk-size-gb", 0, "The disk size to use when creating the image in GB. Value of '0' "+
 		"indicates the default size.")
 	flags.DurationVar(&f.timeout, "timeout", time.Hour, "Timeout value of the image build process. Must be formatted "+
 		"according to Golang's time.Duration string format.")
 }
 
 func (f *FinishImageBuild) validate() error {
+	const IMGSIZE = 10
+	if f.oemSize != "" {
+		oemSizeGB, err := partutil.ConvertSizeToGBRoundUp(f.oemSize)
+		if err != nil {
+			return fmt.Errorf("invalid format of oem-size: %s, error msg:(%v)", f.oemSize, err)
```

Comment:
nit: prefer to use %q instead of %s when printing string variable values. It typically makes the result a little easier to read.